README.md

```diff
@@ -71,6 +64,367 @@ Connected (press CTRL+C to quit)
 ...
 ```
 
+<!-- cSpell: enable -->
+
 ## UI explorer
 
-There is a handly UI for this project called [blockfrost-websocket-link-ui](https://github.com/blockfrost/blockfrost-websocket-link-ui) and you can find its hosted version at [websocket-link.blockfrost.dev](https://websocket-link.blockfrost.dev/).
+There is an example UI for this project called [blockfrost-websocket-link-ui](https://github.com/blockfrost/blockfrost-websocket-link-ui)
+and you can find its hosted version at [websocket-link.blockfrost.dev](https://websocket-link.blockfrost.dev/).
+
+## Documentation
+
+### Configuration
+
+`blockfrost-websocket-link` can be configured through environment variables.
+
+#### Required
+
+- `BLOCKFROST_NETWORK`: let you choose which network it should connect to (_mainnet_ or _testnet_)
+- `BLOCKFROST_PROJECT_ID`: **Blockfrost.io**'s project token
+
+#### Optional
+
+- `BLOCKFROST_BACKEND_URL`: URL pointing to your own backend (`blockfrost-backend-ryo`) if you prefer not to use the
+public **Blockfrost REST API**
+- `BLOCKFROST_BLOCK_LISTEN_INTERVAL`: how often should be the server pulling the backend for new data (in milliseconds,
+default `5000`)
+- `BLOCKFROST_FIAT_RATES_PROXY`: the proxy used to fetch fiat rates
+- `BLOCKFROST_SENTRY_DSN`: the **Sentry** data source name to optionally monitor the service
+- `BLOCKFROST_WSLINK_DEBUG`: enables `debug` logging level
+- `BUILD_COMMIT`: provided by `/status` endpoint
+- `METRICS_COLLECTOR_INTERVAL_MS`: frequency for refreshing metrics and performing health check (default `10000`)
+- `PORT`: which port the server should listen to (default `3005`)
+
+### API
+
+As its name suggests, `blockfrost-websocket-link` offers a web socket based API: the client needs to connect to it
+through a web socket. Once the web socket connection is correctly established the server is immediately ready to accept
+commands. The communication (both ways) is _JSON encoded_: the server expects JSON encoded messages and will respond
+using the same encoding.
+
+Each input message (also known as command) requires a nonce which is mirrored by the server in the output message; it
+is not actually used by the server but it useful for the client to reconcile output messages to relative input message.
+
+Each input message requires a command as well (details will follow) and optionally a set of parameters, depending on
```

Comment:
> Each input message (also known as command)

> Each input message requires a command as well

Since we use `command` field in the message I wouldn't call the message itself as command, it creates a a bit of a confusion.

Reply:
be54e33 should solve this